dagger-core/src/main/java/io/odpf/dagger/core/processors/internal/InternalPostProcessor.java

```diff
@@ -60,7 +64,7 @@ public StreamInfo process(StreamInfo streamInfo) {
     protected StreamDecorator getInternalDecorator(InternalSourceConfig internalSourceConfig, ColumnNameManager columnNameManager) {
         SqlConfigTypePathParser sqlPathParser = new SqlConfigTypePathParser(internalSourceConfig, columnNameManager);
         InternalConfigProcessor processor = InternalConfigHandlerFactory
-                .getProcessor(internalSourceConfig, columnNameManager, sqlPathParser);
+                .getProcessor(internalSourceConfig, columnNameManager, sqlPathParser, configuration);
```

Comment:
adding this extra config as a param doesn't seem right. Can we not put the schema class name in the internal source config? That's what we are using?

Reply:
Sure, will change it - I'd seen that with transformers we have both a separate transformer config argument and this configuration argument, so I'd tried keeping that structure here.

Reply:
Yes, but the configuration is not needed here. @prakharmathur82 can you comment?

Reply:
Yeah will remove the `configuration` argument if we go ahead with keeping schema class name in internal source config.

Reply:
As discussed with @prakharmathur82, will introduce an optional `functionProcessorConfig` argument (tentative name) as part of `internalSourceConfig`, which will be a `Map<String, String>` for now, and schema class name will be a part of that optional config.

```
PROCESSOR_POSTPROCESSOR_CONFIG={
  "internal_source": [
    {
      "output_field": "payload",
      "value": "JSON_PAYLOAD",
      "type": "function",
      "function_processor_config": {
        "schema_proto_class": "com.foo.bar.RestaurantMessage"
      }
    }
  ]
}
```